elements/lisk-bft/src/utils.ts

```diff
@@ -15,25 +15,11 @@
 import { validator } from '@liskhq/lisk-validator';
 
 import { blockHeaderSchema } from './schema';
-import { Block, BlockHeader } from './types';
+import { BlockHeader } from './types';
 
-export const validateBlockHeader = (blockHeader: BlockHeader): boolean => {
+export const validateBlockHeader = (blockHeader: BlockHeader): void => {
```

Comment:
I think this function is just wrapper around `validator.validate` maybe its better to use it directly instead of adding as a util func